### ISSUE TITLE
"Lonely Turkey" summoned by "Turkey Caller" missing scripting

### ISSUE THREAD

**Issue by Shineslippers** [azerothcore/azerothcore-wotlk#8817]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour

[Lonely Turkey](https://wowgaming.altervista.org/aowow/?npc=32956) summoned by [Turkey Caller](https://wowgaming.altervista.org/aowow/?item=44844) missing on spawn script, on [Turkey Shooter](https://wowgaming.altervista.org/aowow/?item=44812) script.

https://user-images.githubusercontent.com/85256266/139445404-6ee4bf09-9f0a-44e7-800b-8c65c158b818.mp4

### Expected Blizzlike Behaviour

After summmon turkey have to summonet with heart and after run out of you with broken heart. Also same script have be for Turkey Shooter targets. 
![192738](https://user-images.githubusercontent.com/85256266/139443230-3a29c89a-452e-4d25-b59b-c9461248a936.jpg)

### Source

Wowhead [Turkey Caller](https://www.wowhead.com/item=44844/turkey-caller#comments:id=897333)
> By Chancakes on 2009/11/22 (Patch 3.2.2)
Using this item will spawn a Lonely Turkey **neutral mob that will stand near you with a heart over its head**. It serves no apparent purpose, but you can kill it for a Wild Turkey which you can use towards the Pilgrim's Bounty cooking recipe.

> By 105913 on 2009/11/23 (Patch 3.2.2) 
i found out today while waiting for a friend to come with the rest of the grp to H UP that if theres a lonely turkey out **when u turkey shoot a rogue it gets a heart above its head and runs to the rogue**

Wowwiki [Turkey Caller](https://wowwiki-archive.fandom.com/wiki/Turkey_Caller?oldid=2129787)
>Using it will spawn a Combat Lonely Turkey that runs to you, then has a broken heart and runs off until you kill it.

### Steps to reproduce the problem

.additem 44844
use it

### Extra Notes

Hope i will find some videos later.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/b0d4f1917ec82d978121859b2ea45cdc178bb478

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by UltraNix** [comment]
@Shineslippers Yep - I need some video about it, because I'm not sure how it should look like.
So player summons that turkey with heart above. Then run to that player, get broken heart and then what? Should wander or just follow the player?

**Comment by Shineslippers** [comment]
Best description what i can find [comment](https://www.wowhead.com/npc=32956/lonely-turkey#comments:id=897874)
> By Indridcold on 2009/11/22 (Patch 3.2.2)
Once the Lonely Turkey is summoned he will wonder around for 2 minutes. After the 2 minutes are up it will simply fade away. Technically you could have one around all the time because the cool down for the Turkey Caller is also 2 minutes, unless you kill it that is.

Still no videos around.

**Comment by UltraNix** [comment]
This comment is not really clear to me.

**Comment by Shineslippers** [comment]
Well, i can't do more. Searching did not get results.
